model/src/main/kotlin/SnippetFinding.kt

```diff
@@ -17,9 +17,9 @@
  * License-Filename: LICENSE
  */
 
-package org.ossreviewtoolkit.model.utils
+package org.ossreviewtoolkit.model
 
-import org.ossreviewtoolkit.model.TextLocation
+import org.ossreviewtoolkit.model.utils.Snippet
```

Comment:
Argh, sorry, IMO `Snippet` should be moved as well, as it's not a utility to work on model classes, but a model class itself.